Parsers/ASimNetworkSession/Parsers/vimNetworkSessionCrowdStrikeFalconHost.yaml

```diff
@@ -0,0 +1,379 @@
+                        "networkProfile", networkProfile,
+                        "ruleDescription", ruleDescription,
+                        "ruleGroupName", ruleGroupName,
+                        "cmdLine", cmdLine
+                    ),
+              NetworkIcmpCode = icmpCode
+          | invoke _ASIM_ResolveDvcFQDN('hostName')
+          | invoke _ASIM_ResolveNetworkProtocol('protocol')
+          | extend NetworkIcmpType = _ASIM_LookupICMPType('icmpType')
+          | project-rename
+              DvcId = deviceId,
+              DvcIpAddr = deviceIp,
+              EventOriginalSubType = eventType,
+              NetworkRuleName = ruleName
+          | extend
+              Rule = NetworkRuleName,
+              Dvc = coalesce(DvcId, DvcHostname, DvcIpAddr);
+      let networkaccessdata = alldata
+          | where DeviceEventClassID has "Network Access In A Detection Summary Event"
+          | lookup ActionLookup on EventOutcome
+          | where ((array_length(dvcaction) == 0) or DvcAction has_any (dvcaction))
+          | where ((eventresult == "*") or EventResult == eventresult)
+          | extend
+              temp_isSrcMatch = has_any_ipv4_prefix(SourceIP, src_or_any), 
+              temp_isDstMatch = has_any_ipv4_prefix(DestinationIP, dst_or_any)
+          | extend ASimMatchingIpAddr = case(
+                                    array_length(src_or_any) == 0 and array_length(dst_or_any) == 0,
+                                    "-", 
+                                    (temp_isSrcMatch and temp_isDstMatch),
+                                    "Both", 
+                                    temp_isSrcMatch,
+                                    "SrcIpAddr",
+                                    temp_isDstMatch,
+                                    "DstIpAddr",
+                                    "No match" 
+                                )        
+          | where ASimMatchingIpAddr != "No match"
+          | parse-kv AdditionalExtensions as (CSMTRPatternDisposition: string) with (pair_delimiter=";", kv_delimiter="=")
+          | invoke _ASIM_ResolveSrcFQDN('DestinationHostName')
+          | extend ASimMatchingHostname = case(
+                                      array_length(hostname_has_any) == 0,
+                                      "-",
+                                      SrcHostname has_any (hostname_has_any),
+                                      "SrcHostname",
+                                      "No match"
+                                  )
+          | where ASimMatchingHostname != "No match"
+          | extend
+              EventStartTime = todatetime(DeviceCustomDate1),
+              DstIpAddr = coalesce(DestinationIP, DeviceCustomIPv6Address3),
+              SrcIpAddr = coalesce(SourceIP, DeviceCustomIPv6Address2),
+              EventCount = int(1),
+              EventType = "EndpointNetworkSession",
+              SrcDomain = coalesce(SrcDomain, DestinationNTDomain),
```

Comment:
DestinationNTDomain is not Dst ?

Reply:
it is the Source Device Domain value where the CrowdStrike agent is installed and the event is reported.